opm/parser/eclipse/EclipseState/Runspec.hpp

```diff
@@ -80,6 +80,31 @@ class Welldims {
     int nGMax  { 0 };
 };
 
+class WellSegmentDims {
+public:
+    explicit WellSegmentDims(const Deck& deck);
+
+    int maxSegmentedWells() const
+    {
+        return this->nSegWellMax;
+    }
+
+    int maxSegmentsPerWell() const
+    {
+        return this->nSegmentMax;
+    }
+
+    int maxLateralBranchesPerWell() const
+    {
+        return this->nLatBranchMax;
+    }
+
+private:
+    int nSegWellMax   { 0 };
```

Comment:
I would prefer if the defaults were fetched from the Json scheme like here: https://github.com/OPM/opm-common/blob/35ebab0b69c6a125ea350a75ea1b4ceb359294ab/opm/parser/eclipse/EclipseState/Tables/Eqldims.hpp#L34

Reply:
> I would prefer if the defaults were fetched from the Json scheme

Very good.  I'll update accordingly.

Reply:
>> I would prefer if the defaults were fetched from the Json scheme
>
> Very good. I'll update accordingly.

Please see commit aa97b76.